pyHalo/realization_extensions.py

```diff
@@ -206,6 +210,65 @@ def add_core_collapsed_halos(self, indexes, **kwargs_halo):
                                       rendering_center_x, rendering_center_y,
                                       self._realization.geometry)
 
+    def add_ULDM_fluctuations(self,de_Broglie_wavelength,amplitude_std_dev,gaussian_std_dev,shape='ring',args={'rmin':0.9,'rmax':1.1}):
```

Comment:
Could you change the arguments to have a more clear physical interpretation, something like fluctuation_amplitude_variance instead of amplitude_std_dev, and fluctuation_size_variance instead of gaussian_std_dev?

Reply:
When you say fluctuation_amplitude_variance do you actually mean the variance (delta Sigma)^2, or are you just calling the delta Sigma the variance?

Reply:
I guess it's the square root of the variance, or delta_kappa in the overleaf, i.e. sqrt(< \delta k>^2), so that you sample from np.random.normal(0, delta_kappa)